snapcraft/plugins/v2/make.py

```diff
@@ -46,15 +46,7 @@ def get_schema(cls) -> Dict[str, Any]:
             "$schema": "http://json-schema.org/draft-04/schema#",
             "type": "object",
             "additionalProperties": False,
-            "properties": {
-                "make-parameters": {
```

Comment:
i thought we were going to maintain the schema where practical?

Reply:
this piece makes no sense though